index.js

```diff
@@ -34,7 +34,10 @@ module.exports = function (linkHeader) {
     .map(parseLink)
     .filter(function (x) { return x && x.rel; })
     .reduce(function (acc, x) {
-      acc[x.rel] = x;
+      x.rel.split(/\s+/).forEach(function (rel) {
```

Comment:
Yeah pull them to the highest scope possible, but as long as they aren't inside the `forEach` declaration it's all good.
However we don't have to worry about this right now, i.e. we can take care of this in another PR.